source/docs/getting-started/adding-platform.html.md

```diff
@@ -0,0 +1,194 @@
+       Preparing...                          ################################# [100%]
+       Updating / installing...
+          1:chef-13.0.118-1.el7              ################################# [100%]
+       Thank you for installing Chef!
+       Transferring files to <default-centos-73>
+       Starting Chef Client, version 13.0.118
+       Creating a new client identity for default-centos-73 using the validator key.
+       resolving cookbooks for run list: ["git_cookbook::default"]
+       Synchronizing Cookbooks:
+         - git_cookbook (0.1.0)
+       Installing Cookbook Gems:
+       Compiling Cookbooks...
+       Converging 2 resources
+       Recipe: git_cookbook::default
+         * yum_package[git] action install
+           - install version 1.8.3.1-6.el7_2.1 of package git
+         * log[Well, that was too easy] action write
+
+
+       Running handlers:
+       Running handlers complete
+       Chef Client finished, 2/2 resources updated in 12 seconds
+       Finished converging <default-centos-73> (0m27.28s).
+-----> Setting up <default-centos-73>...
+       Finished setting up <default-centos-73> (0m0.00s).
+-----> Verifying <default-centos-73>...
+       Loaded tests from test/smoke/default
+
+Profile: tests from test/smoke/default
+Version: (not specified)
+Target:  ssh://vagrant@127.0.0.1:2222
+
+
+  System Package
+     ✔  git should be installed
+
+Test Summary: 1 successful, 0 failures, 0 skipped
+       Finished verifying <default-centos-73> (0m1.52s).
+-----> Kitchen is finished. (1m11.96s)
+~~~
+
+Nice! We've verified that our cookbook works on Ubuntu 16.04 and CentOS 7.3. Since the CentOS instance will hang out for no good reason, let's kill it for now:
+
+~~~
+kitchen destroy
+-----> Starting Kitchen (v1.16.0)
+-----> Destroying <default-ubuntu-1604>...
+       Finished destroying <default-ubuntu-1604> (0m0.00s).
+-----> Destroying <default-centos-73>...
+       ==> default: Forcing shutdown of VM...
+       ==> default: Destroying VM and associated drives...
+       Vagrant instance <default-centos-73> destroyed.
+       Finished destroying <default-centos-73> (0m4.96s).
+-----> Kitchen is finished. (0m6.04s)
+~~~
+
+Interesting. Kitchen tried to destroy both instances, one that was created and the other that was not. Which brings us to another tip with the `kitchen` command:
+
+> **Any `kitchen` subcommand without an instance argument will apply to all instances.**
+
+Let's make sure everything has been destroyed:
+
+~~~
+$ kitchen list
+Instance             Driver   Provisioner  Verifier  Transport  Last Action    Last Error
+default-ubuntu-1604  Vagrant  ChefZero     Inspec    Ssh        <Not Created>  <None>
+default-centos-73    Vagrant  ChefZero     Inspec    Ssh        <Not Created>  <None>
+~~~
+
+<div class="sidebar--footer">
+<a class="button primary-cta" href="/docs/getting-started/dynamic-configuration">Next - Dynamic Configuration</a>
```

Comment:
Should dynamic config really be next? I'd think platform -> the feature flow -> dynamic config -> next steps?